build.sh

```diff
@@ -222,7 +222,7 @@ case $__HostOS in
         __monoroot=/usr/local
         ;;
     OSX)
-        __monoroot=/Library/Frameworks/Mono.framework/Versions/Current
+        __monoroot=/usr/local
```

Comment:
Why is this change necessary? Mono is, by default, located in the /Library folder....